### PR TITLE
Add vision to handbook.

### PR DESCRIPTION
**Pull request by tpendragon**
Looking for wordsmithing (on everything but the vision text) or formatting advice here.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,6 +22,17 @@ of the following ways:
 - Create a [Github issue](https://github.com/pulibrary/dls-handbook/issues)
 - Message us in #digital_library in PULibrary's Slack
 
+## Our Vision
+
+Princeton University Library has defined its [Mission, Vision, and North Star
+Statements](https://library.princeton.edu/about) and DLS has worked with its
+partners to define the Digital Library's part in the wider vision and how we can
+best move it forward. It is as follows:
```
Comment (hackartisan) [inline README.md:30]: This sentence is pretty long. I took a stab at breaking it into 2 simpler sentences.

DLS has worked with its partners to define a vision for the Digital Library in the context of Princeton University Library's Mission Vision, and North Star Statements. This Digital Library vision helps us situate our work within the broader vision, and helps us move that vision forward. It is as follows:

File: README.md [hunk 2]
```diff
@@ -22,6 +22,17 @@ of the following ways:
...
+
+Princeton University's Digital Library will become an inspiring environment for global communities to engage with diverse collections. To support this goal, our Digital Library platform will enable easy ingest, long-term preservation, joyful discovery, and transformative use of digital materials.
+
+DLS will prioritize development and processes in such a way to have the most impact towards this vision, and we encourage our partners and stakeholders to imagine ways for us to get there together.
```
Comment (hackartisan) [inline README.md:34]: I like this part. I think it should be "in such a way as to have." Maybe we should expand "development and processes" into, like, "development work and organization processes"

File: README.md (hunk 1 above)
Comment (kelea99) [reply README.md:30]: Remove this from start of sentence, perhaps? and capitalize vision (in line with how the vision is capitalized for PUL)

Our Digital Library Vision  helps us.... or
The Digital  Library Vision helps us...

File: README.md (hunk 2 above)
Comment (kelea99) [reply README.md:34]: I really like expanding it, @hackartisan

File: README.md (hunk 1 above)
Comment (tpendragon) [reply README.md:30]: The draft felt a little duplicative, how does...

```
DLS has worked with its partners to define a vision for the Digital Library in
the context of Princeton University Library's [Mission, Vision, and North Star
Statements](https://library.princeton.edu/about) to help us situate
our work within the broader vision and move it forward. It is as
follows:
```

sound?

File: README.md [hunk 3]
```diff
@@ -22,6 +22,18 @@ of the following ways:
 - Create a [Github issue](https://github.com/pulibrary/dls-handbook/issues)
 - Message us in #digital_library in PULibrary's Slack
 
+## Our Vision
+
+DLS has worked with its partners to define a vision for the Digital Library in
+the context of Princeton University Library's [Mission, Vision, and North Star
+Statements](https://library.princeton.edu/about) to help us situate
```
Comment (hackartisan) [inline README.md:29]: I like this! Since we have "to define... to help" what if we made it "that will help" instead.

**Comment by hackartisan** [comment]
I think the current draft is great. I had 1 last recommendation. Is this something to discuss more? Are we otherwise blocked @tpendragon?

**Comment by tpendragon** [comment]
@hackartisan Did I do your suggestion right?

**Review by hackartisan — APPROVED** [review]
👍🏻 looks good!

**Pull request merged**